tools/pythonpkg/src/pyrelation.cpp

```diff
@@ -32,8 +32,8 @@ void DuckDBPyRelation::Initialize(py::handle &m) {
 	         py::arg("other_rel"))
 	    .def("join", &DuckDBPyRelation::Join,
 	         "Join the relation object with another relation object in other_rel using the join condition expression "
-	         "in join_condition",
-	         py::arg("other_rel"), py::arg("join_condition"))
+	         "in join_condition. Types supported are 'inner' and 'left'",
+	         py::arg("other_rel"), py::arg("condition"), py::arg("type") = "left")
```

Comment:
Maybe default to inner?

Reply:
I agree, since `join` implies `inner join` in SQL, it makes sense to reflect that fact in the API here. I think it offers the least surprise :slightly_smiling_face:

Reply:
ah yes that's a mistake